netkan/Dockerfile

```diff
@@ -10,13 +10,13 @@ USER root
 RUN rm -Rf /netkan
 
 FROM python:3.7 as dev
-COPY --from=base /home/netkan /home/netkan
-RUN useradd -Ms /bin/bash netkan
-RUN chown -R netkan:netkan /home/netkan
-ADD run_dev.sh /usr/local/bin/
+RUN useradd -ms /bin/bash netkan
+ADD . /netkan
 WORKDIR /home/netkan
+ADD run_dev.sh /usr/local/bin/
+ENV PATH "$PATH:/home/netkan/.local/bin"
 USER netkan
-ENTRYPOINT ["/usr/local/bin/run_dev.sh"]
+RUN pip install --user /netkan/.[development]
```

Comment:
I know I'm late to the party, but there's missing an entrypoint now, isn't it?
Why was `ENTRYPOINT [".local/bin/netkan"]` removed, or, shouldn't have `ENTRYPOINT [".local/bin/netkan"]` been added after pip?

Reply:
Nah, this is the dev stage. So a combination of docker compose/vs code takes care of this.

Reply:
But in the normal docker-compose the entrypoint is not specified for most services, while targeting the dev stage:
https://github.com/KSP-CKAN/NetKAN-Infra/blob/a048da12cc36d5923314579aff0d83d33ae37040/docker-compose.yml#L7-L27

Reply:
Good point. I haven't used the regular file since moving, the idea was to be able to spin up a whole dev set of services, but need to invest some more time making it all work in dev.